recipes/GaitPy/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "gaitpy" %}
+{% set version = "1.6.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/matt002/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3b0001220a2f1f19df638b3e26f9ab6838104decbf2dabf303aa71d6cdd29c42
+
+build:
+  skip: True  # [py!=36]
+  number: {{ build }}
+  script: "{{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt"
+
+requirements:
+  build:
```

Comment:
Please put all dependencies from `build` to `host` and remove `build`. The `build` section is usually used for compilers, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements-section.